department-of-veterans-affairs/vets-website
# Add medicaid pages

## Summary
On the Pension Benefits application(21P-527EZ), update the 'Health and employment information' chapter

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Select 'Start your application without signing in'
3. Verify the 'Medicaid coverage' page is in the correct location of the 'Health and employment information' chapter, per the designs, and includes a yes/no radio question
4. Verify that 'Yes' takes you to the 'Special monthly pension' page
5. Verify that 'No' takes you to the 'Medicaid application status' page
6. On the 'Medicaid application status' page, verify that an alert displays when 'Yes' is selected
7. Verify that an alert is not displayed when 'No' is selected
8. Verify that all changes are saved to the redux store in form data and included in the save form api call

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69751)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

## Screenshots

### Desktop
| Medicaid coverage | Medicaid application status | Medicaid application status (Alert) |
| ------ | ------ | ------ | 
|![localhost_3001_pension_application_527EZ_medical_history_nursing_medicaid_status](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/a92661a9-27b5-497d-83c2-8caf04dfd5b0)|![localhost_3001_pension_application_527EZ_medical_history_nursing_medicaid_status (1)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/63447a3a-864a-4120-bafc-15952e9ad94e)|![localhost_3001_pension_application_527EZ_medical_history_nursing_medicaid_status (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/1d4d446d-f6e5-4ada-860e-1586bf5b3eca)

## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69751)

### Quality Assurance & Testing

### Error Handling


### Authentication
